colabs/Wczytywanie_danych_NASA_w_sparku.ipynb

```diff
@@ -0,0 +1,262 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "provenance": [],
```

Comment:
`
if column == "Date":
        df = df.withColumn(column, col(column).cast(IntegerType()))
`
To będzie prowadzić do sporych wypaczeń, bo arytmetyka dat jest inna niż integera. Lepiej byłoby pociąć te łańchuchy datowe i zrobić z nich dwie kolumny - rok oraz miesiąc